tests/table.spec.ts

```diff
@@ -1,16 +1,16 @@
 import { test, expect, Page } from '@playwright/test';
-import { generateVarName, pasteInMonaco } from './utils';
+import { makeTableCommand, pasteInMonaco } from './utils';
 
 // Run tests serially since they all use the same table
 test.describe.configure({ mode: 'serial' });
 
-async function openSimpleTable(page: Page) {
+async function openSimpleTable(browserName: string, page: Page) {
```

Comment:
I think it's possible to get this info from the `Page`, but this is fine too

`page.context().browser()?.browserType().name()` gets the name

Reply:
I was trying to figure that out, sweet. I'm going to clean that up.